### PR TITLE
Development

### PR DESCRIPTION
**Pull request by nahukas**

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+.footer-top div {
+  display: flex;
+  flex: 1;
+  cursor: pointer;
+  border: 1.5px solid white;
+
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-color: #464646;
```
Comment (Redvanisation) [inline styles.css]: You can remove the gray background by deleting the background-color property.

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @nahukas 

**Good work on this project, but we need to fix the issues mentioned below before moving on ⬇️** 

1. The **4 bottom links are part of the main content** and not the footer, the footer should include only the copyright, terms of use, and privacy policy links.

2. The **4 bottom links have a gray background color** that it shouldn't be there, please check my comment below to know how to fix that.

3. **Delete commented code** if you need to review a previous version of your code, use git history.f you need to review a previous version of your code, use git history.

4. **Activate Stickler on your repo**, if you don't know how to do that check this [link](https://github.com/microverseinc/linters-config/tree/master/css).

5. Add a **description** to your pull request.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Comment by nahukas** [comment]
Finished changes required by code reviewer.

**Review by Forison — COMMENTED** [review]
# Pull request under review by Addo

File: index.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                src="./assets/img/playButton.png"
+                alt="play logo"
+              />
+            </a>
+          </p>
+        </section>
+      </section>
+      <section class="footer-top">
+        <div class="promo"><a href="#"></a></div>
+        <div class="promo"><a href="#"></a></div>
```
Comment (Forison) [inline index.html]: ```<a href="#"> <div class="promo" ></div></a>```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        <div class="promo"><a href="#"></a></div>
+        <div class="promo"><a href="#"></a></div>
```
Comment (Forison) [inline index.html]: ```<a href="#"> <div class="promo" ></div></a>```

**Review by Forison — CHANGES_REQUESTED** [review]
Hi Nahuel 👋

I like your usage of semantic HTML.🙂🙂🙂 && the ouput of your project. Your navbar looks 👌👌 👌.
You have done an amazing 👌👌 job with this project. I nearly approved it, but you missed one [tiny] detail:
- Each of the four elements under the main is supposed to be links as it stands now they are not links.🙂
> wrap the anchor tag rather around the div with class name ```.promo```

- You should fix the [warning] given to you by the validator when uploading the index file.🙂
>try using ```div``` instead of the section where you have those warnings occurring.

### That is it Nahuel,
### Kindly do these changes and submit for review
happy coding 👋👋

**Review by Forison — COMMENTED** [review]
# pull request under review by ADDO

**Review by Forison — APPROVED** [review]
# 👋👋👋👋Good Job Nahuel 👋👋👋👋👋
### 👌Project looks great👌
### You can go ahead and merge to master, once again, good Job👋👋👋👋
Happy coding

**Pull request merged**